### PR TITLE
Re organise

### PR DESCRIPTION
**Pull request by jburel**
Re-organise the landing page and simplify header

### CODE REVIEW

File: docs/manual_analysis.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Analyze manually OMERO data using the OMERO plugin for Fiji/ImageJ
-==================================================================
+Analyze, save ROIS and measurement
```
Comment (pwalczysko) [inline docs/manual_analysis.rst]:
```suggestion
Analyze, save ROIs and measurements
```

File: docs/manual_analysis.rst [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+==================================
```
Comment (pwalczysko) [inline docs/manual_analysis.rst]:
```suggestion
===================================
```

File: docs/imagej_python.rst [hunk 3]
```diff
@@ -49,53 +43,15 @@ Install omero-py and `pyimagej <https://pypi.org/project/pyimagej/>`_ via Conda:
 
 Note: we have noticed problems using the Option 1 approach when running the scripts on some operating systems due to issues with the ImageJ1-ImageJ2 bridge.
```
Comment (pwalczysko) [inline docs/imagej_python.rst]:
```suggestion
Note: we have noticed problems using the Conda approach when running the scripts on some operating systems due to issues with the ImageJ1-ImageJ2 bridge.
```

**Comment by pwalczysko** [comment]
Reviewing the python&Fiji chapter:

- ``conda create -n imagej_python python=3.6`` does not work on M1 - also, 3.6 is old version, I would replace the cmd with python=3.8 (which worked on M1 for the install, also the following steps)
- after a successful install, I am getting probs with 
   - interactive mode for Fiji is not available on Mac (solved by setting ``headless=True``)
   - the script crashes on 
 
```
File "/Users/pwalczysko/Downloads/run_macro_python.py", line 80, in analyse
    from jnius import autoclass
ModuleNotFoundError: No module named 'jnius'
```

When I installed ``pyjnius`` (as per https://stackoverflow.com/questions/59745726/how-to-install-jnius-in-python the jnius seems to be deprecated in favor of pyjnius), and corrected the script to point to pyjnius, I still get an error "no module named pyjnius"

**Comment by jburel** [comment]
@pwalczysko I am planning to review that in a follow-up PR
and add a GHA workflow to test it

**Comment by pwalczysko** [comment]
> @pwalczysko I am planning to review that in a follow-up PR and add a GHA workflow to test it

Great, thanks. If it is any help, I have also run (successfully) the build of this PR on MyBinder.
After that, I failed to find the place where I can start a new Desktop - instead, run from the inside of the terminal:
Edit: I have found teh place where to start a new Desktop, started it, then run from the terminal the script and this failed with the ``junius`` error, just like in the case on local M1:

```
jovyan@jupyter-jburel-2domero-2dguide-2dfiji-2dl5ecp8mf:~/scripts/python$ python run_macro_python.py 
Host [wss://idr.openmicroscopy.org/omero-ws]:                          
Image ID [1884807]: 
initializing fiji...
The headless flag of imagej.init is deprecated. Use the mode argument instead.
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/srv/conda/vnc/Fiji.app/jars/logback-classic-1.2.12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/srv/conda/vnc/Fiji.app/plugins/omero_ij-5.6.1-all.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [ch.qos.logback.classic.util.ContextSelectorStaticBinder]
connecting to IDR...
running the macro...
Traceback (most recent call last):
  File "/home/jovyan/scripts/python/run_macro_python.py", line 106, in <module>
    main()
  File "/home/jovyan/scripts/python/run_macro_python.py", line 99, in main
    analyse(ij, conn, image_id)
  File "/home/jovyan/scripts/python/run_macro_python.py", line 80, in analyse
    from jnius import autoclass
ModuleNotFoundError: No module named 'jnius'
jovyan@jupyter-jburel-2domero-2dguide-2dfiji-2dl5ecp8mf:~/scripts/python$ 
```

Edit 2: It is enough to just ``pip install pyjnius`` and leave the script as it is, but then, the script crashes later with

```
ovyan@jupyter-jburel-2domero-2dguide-2dfiji-2dl5ecp8mf:~/scripts/python$ pip install pyjnius
Collecting pyjnius
  Obtaining dependency information for pyjnius from https://files.pythonhosted.org/packages/62/c9/6ae043600ddeae09376fe60c09d1b39265991f811bdab58a9218c3aab6ae/pyjnius-1.5.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata
  Downloading pyjnius-1.5.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (10 kB)
Downloading pyjnius-1.5.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.4 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.4/1.4 MB 16.2 MB/s eta 0:00:00
Installing collected packages: pyjnius
Successfully installed pyjnius-1.5.0
jovyan@jupyter-jburel-2domero-2dguide-2dfiji-2dl5ecp8mf:~/scripts/python$ python run_macro_python.py 
Host [wss://idr.openmicroscopy.org/omero-ws]:                          
Image ID [1884807]: 
initializing fiji...
The headless flag of imagej.init is deprecated. Use the mode argument instead.
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/srv/conda/vnc/Fiji.app/jars/logback-classic-1.2.12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/srv/conda/vnc/Fiji.app/plugins/omero_ij-5.6.1-all.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [ch.qos.logback.classic.util.ContextSelectorStaticBinder]
connecting to IDR...
running the macro...
Traceback (most recent call last):
  File "/home/jovyan/scripts/python/run_macro_python.py", line 106, in <module>
    main()
  File "/home/jovyan/scripts/python/run_macro_python.py", line 99, in main
    analyse(ij, conn, image_id)
  File "/home/jovyan/scripts/python/run_macro_python.py", line 80, in analyse
    from jnius import autoclass
  File "/srv/conda/envs/notebook/lib/python3.10/site-packages/jnius/__init__.py", line 45, in <module>
    from .reflect import *  # noqa
  File "/srv/conda/envs/notebook/lib/python3.10/site-packages/jnius/reflect.py", line 19, in <module>
    class Class(JavaClass, metaclass=MetaJavaClass):
  File "jnius/jnius_export_class.pxi", line 118, in jnius.MetaJavaClass.__new__
  File "jnius/jnius_export_class.pxi", line 178, in jnius.MetaJavaClass.resolve_class
  File "jnius/jnius_env.pxi", line 11, in jnius.get_jnienv
  File "jnius/jnius_jvm_dlopen.pxi", line 95, in jnius.get_platform_jnienv
  File "jnius/jnius_jvm_dlopen.pxi", line 87, in jnius.create_jnienv
SystemError: JVM failed to start: -5
```

**Comment by pwalczysko** [comment]
Opened https://github.com/jburel/omero-guide-fiji/pull/2 to fix some discrepancies of plugin names in Fiji

**Comment by pwalczysko** [comment]
Happy with this PR after https://github.com/ome/omero-guide-fiji/pull/30#issuecomment-1638275128 is merged.

**Review by pwalczysko — APPROVED** [review]

**Pull request merged**